modules/custom/az_migration/migrations/migrate_plus.migration.az_media.yml

```diff
@@ -65,7 +65,13 @@ process:
         plugin: default_value
```

Comment:
In this file we need to change the bundle of videos based on filemime. If oembed/video, make the bundle az_remote_video, otherwise az_video. See #1521